recipes/hwloc/all/conanfile.py

```diff
@@ -0,0 +1,120 @@
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rm, rmdir
+from conan.tools.gnu import Autotools, AutotoolsDeps, AutotoolsToolchain
+from conan.tools.layout import basic_layout
+import os
+
+required_conan_version = ">=1.53.0"
+
+
+class HwlocConan(ConanFile):
+    name = "hwloc"
+    description = "Portable Hardware Locality (hwloc)"
+    topics = ("hwloc")
+    license = "The 2-Clause BSD License"
```

Comment:
I think that it is closer to BSD-3-Clause

https://www.open-mpi.org/projects/hwloc/license.php
https://spdx.org/licenses/BSD-3-Clause.html

```suggestion
    license = "BSD-3-Clause"
```

Reply:
thanks, fixed!